contracts/hardhat.config.ts

```diff
@@ -71,6 +71,10 @@ let config = {
         version: '0.8.19',
         settings: COMPILER_SETTINGS,
       },
+      {
+        version: '0.8.20',
```

Comment:
Doesn't feel needed as it uses 0.8.19?

Reply:
It seems like it's needed because it's fixed to 0.8.20 here in the dependency: https://github.com/matter-labs/era-contracts/blob/main/system-contracts/contracts/libraries/EfficientCall.sol

Reply:
That's 20 and up, we don't really use 20 at CLL, we skip to 24. I'd recommend you bump everything to 24 to not introduce yet another Solidity version. Many projects are already on 24/26 so it shouldn't be an issue to bump Functions.

Reply:
@RensR Sorry forgot to mention that additionally there's a weird thing where the versions are fixed / strict once they're imported via npm. So basically EfficientCall on github is ^0.8.20 but if you check it within node_modules it's strictly 0.8.20. 
It seems to be an open issue: https://github.com/matter-labs/era-contracts/issues/1209